pmm/aws-staging-start-pmm3.groovy

```diff
@@ -0,0 +1,486 @@
+                                        fi
+                                        docker network create pmm-qa || true
+
+                                        docker run -d \
+                                            -p 80:80 \
+                                            -p 443:443 \
+                                            -p 9000:9000 \
+                                            --volumes-from ${VM_NAME}-data \
+                                            --name ${VM_NAME}-server \
+                                            --network pmm-qa \
+                                            --restart always \
+                                            $ENV_VARIABLE \
+                                            ${DOCKER_VERSION}
+
+                                        sleep 10
+                                        docker logs ${VM_NAME}-server
+
+                                        if [ ${ADMIN_PASSWORD} != admin ]; then
+                                            if [ ${CHANGE_USER_PASSWORD_UTILITY} == yes ]; then
+                                                docker exec ${VM_NAME}-server change-admin-password ${ADMIN_PASSWORD}
+                                            else
+                                                docker exec ${VM_NAME}-server grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password ${ADMIN_PASSWORD}
+                                            fi
+                                        fi
+                                    fi
+                                '''
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        stage('Run version service') {
+            when {
+                expression { env.VERSION_SERVICE_IMAGE != "" }
+            }
+            steps {
+                script {
+                    withEnv(['JENKINS_NODE_COOKIE=dontKillMe']) {
+                        node(env.VM_NAME){
+                            sh """
+                                set -o errexit
+                                set -o xtrace
+                                docker run --name ${VM_NAME}-version-service -d --hostname=${VM_NAME}-version-service -e SERVE_HTTP=true -e GW_PORT=80 ${VERSION_SERVICE_IMAGE}
+                                docker network create ${VM_NAME}-network
+                                docker network connect ${VM_NAME}-network ${VM_NAME}-version-service
+                                docker network connect ${VM_NAME}-network ${VM_NAME}-server
+                            """
+                        }
+                    }
+                }
+            }
+        }
+        stage('Enable Testing Repo') {
+            when {
+                expression { env.ENABLE_TESTING_REPO == "yes" && env.PMM_VERSION == "pmm2" && env.CLIENT_INSTANCE == "no" }
```

Comment:
@BupycHuk Should this also be changed to
` expression { env.ENABLE_TESTING_REPO == "yes" && env.PMM_VERSION == "pmm3" && env.CLIENT_INSTANCE == "no" }`

Reply:
not sure, better to ask @puneet0191. I'm not even sure that we need to have separate job for pmm3, because as far as I see `aws-staging-start` was designed the way to support different PMM options. 
@puneet0191 your thoughts?

Reply:
yes, we don't need to have a different staging job for pmm3, we can use the same one, just adding pmm3 as another version, we might refactor pmm-framework.sh or have something completely new for pmm3, but that can still be utilized from the same job.

Reply:
@surbhat1595 let's remove aws-staging-start-pmm3 for now and use old aws-staging-start for now.